bin/prod-deploy/aws.sh

```diff
@@ -215,14 +230,13 @@ function waitForInstanceToBeReady() {
 #
 # $1 - ID of the EC2 instance to wait for
 #
-# Expects global environment variables:
-#   AWS_PROD_API_PORT - Port the EC2 instance is listening on
-#   AWS_PROD_API_TARGET_GROUP_ARN - The ARN of the target group to remove the
-#     instance from.
+# Expects global variables:
+#   API_PORT - Port the EC2 instance is listening on
+#   AWS_TARGET_GROUP - The ARN of the target group to remove the instance from.
 function waitForTargetToBeHealthy() {
   HEALTH_CHECK_COUNT=1
   TARGET_HEALTH=$(getTargetHealth $1)
-  while [[ "$TARGET_HEALTH" != "healthy" && "HEALTH_CHECK_COUNT" -lt 10 ]]; do
+  while [[ "$TARGET_HEALTH" != "healthy" && "HEALTH_CHECK_COUNT" -lt 60 ]]; do
```

Comment:
Guessing that bumping up the health check count gives us more retries?

Reply:
Yeah. This is probably overkill, but it turns out that *every now and then* it takes more than 10 minutes for a machine to start responding. No idea why. Probably gremlins.

Reply:
![satan](https://66.media.tumblr.com/392ea9a32f8162823d22a39f382e85c0/tumblr_noa00amyD31u5gaxgo1_500.gif)

Reply:
![well damn](https://media.giphy.com/media/k8CmOMUgo8uGs/giphy.gif)